source/process/definitions.rst

```diff
@@ -0,0 +1,64 @@
+.. _definitions:
+
+Levels of Feature Quality, Development, and Support
```

Comment:
Instead of "Support" propose "Support Eligibility Level" as we're talking about the potential support available, not the support promised.